CMakeLists.txt

```diff
@@ -97,6 +97,9 @@ endif()
 if(EXISTS "${OpenVINOGenAI_SOURCE_DIR}/tests/cpp")
```

Comment:
```suggestion
if(EXISTS "${OpenVINOGenAI_SOURCE_DIR}/tests")
```
and add `CMakeLists.txt` to `tests`

Reply:
Why is this check is needed in the first place? These folders are always here because we put add them

Reply:
done